frontend/src/components/Logout/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FunctionComponent } from 'react';
+import { connect } from 'react-redux';
+import { Routine } from 'redux-saga-routines';
+import { Button } from 'react-bootstrap';
+import { history } from '../../common/helpers/historyHelper';
+import { Routes } from '../../common/enums/Routes';
+import { logoutUserRoutine } from '../../routines/user';
+import { ITokens } from '../../common/models/ITokens';
+import { setTokens, getRefreshToken } from '../../common/helpers/storageHelper';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSignOutAlt } from '@fortawesome/free-solid-svg-icons';
+import { logout } from '../../services/authService';
+
+interface IProps {
+  logoutUser: Routine;
+}
+
+export const Logout: FunctionComponent<IProps> = ({ logoutUser }) => {
+  const onLogout = () => {
+    logout(getRefreshToken());
+    const tokens: ITokens = { accessToken: '', refreshToken: '' };
+    setTokens(tokens);
```

Comment:
why don't you create removeTokens method and it will remove tokens from storage?

Reply:
Done